packages/derive/src/callable_point.rs

```diff
@@ -86,23 +90,28 @@ pub fn make_callable_point(function: syn::ItemFn) -> TokenStream {
     } else {
         quote! { let deps = cosmwasm_std::make_dependencies() }
     };
+    (
+        quote! {
+            #[cfg(target_arch = "wasm32")]
```

Comment:
About "wasm32".
Don't you think it's better to declare the character strings used for judgment together like this?

"Deps"
"Deps Mut"
"callable_point"
The same can be said for , so I'll leave it to you. Please advise if it is better to refactor in another PR.